### PR TITLE
samples: nrf_desktop: Start ble_adv when ready and needed

### PR DESCRIPTION
**Pull request by pdunaj**
Jira:DESK-428

Signed-off-by: Pawel Dunaj <pawel.dunaj@nordicsemi.no>

### CODE REVIEW

**Comment by pdunaj** [comment]
@kapi-no , can you have a look?

**Comment by pdunaj** [comment]
added one commit - wake up should be triggered only when system was down

File: samples/nrf_desktop/src/modules/ble_adv.c [hunk 1]
```diff
@@ -34,6 +34,7 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_BLE_ADV_LOG_LEVEL);
 
 
 enum state {
+	STATE_DISABLED,
```
Comment (KAGA164) [inline samples/nrf_desktop/src/modules/ble_adv.c]: If this state can be set anywhere ?

**Comment by pdunaj** [comment]
Hi @pizi-nordic , could you merge?

**Pull request merged**